common/app/controllers/EmailSignupController.scala

```diff
@@ -90,16 +90,36 @@ class EmailSignupController(
       Cached(60)(RevalidatableResult.Ok(views.html.emailLanding(emailLandingPage)))
     }
 
+  def renderFooterForm(listName: String): Action[AnyContent] =
+    csrfAddToken {
+      Action { implicit request =>
+        val id = EmailNewsletter.fromIdentityName(listName).map(_.listIdV1)
```

Comment:
This is carried over from the previous structure: https://github.com/guardian/frontend/pull/23051/files/288c0b3d682749459e773c36f5df8eee198dc7fe#diff-e267b3b11a68744536e7b85541f17293L111
so I don't know the previous motivation. I think this bit only existed to check that the newsletter existed. It can probably be tidied up a bit and made less obscure